package.json

```diff
@@ -23,7 +23,7 @@
     "@azure/data-tables": "13.2.1",
     "@azure/identity": "^4.4.1",
```

Comment:
remove the  ^ we don't want variable version numbers and should be using fixed

Reply:
Fixed